packages/config/src/projects/layer2s/ithaca.ts

```diff
@@ -0,0 +1,26 @@
+import { underReviewL2 } from './templates/underReview'
+import { Layer2 } from './types'
+
+export const ithaca: Layer2 = underReviewL2({
```

Comment:
Ithaca is the name of the company, not the L2. Rename this to Odyssey and other instances of the name here too

Reply:
Thank you for the correction! I’ve made the changes and renamed it to Odyssey in all relevant instances.